docs/api/structures/bounds.md

```diff
@@ -0,0 +1,6 @@
+# Bounds
```

Comment:
I think this might be better name as `Rect` or `Rectangle`.

It is derived from this class, https://cs.chromium.org/chromium/src/ui/gfx/geometry/rect.h